packages/core/drizzle/0082_sparkling_redwing.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE "latitude"."provider_logs" ALTER COLUMN "provider_id" DROP NOT NULL;--> statement-breakpoint
+ALTER TABLE "latitude"."provider_logs" ALTER COLUMN "finish_reason" DROP NOT NULL;--> statement-breakpoint
+ALTER TABLE "latitude"."provider_logs" ALTER COLUMN "config" DROP NOT NULL;--> statement-breakpoint
+ALTER TABLE "latitude"."provider_logs" ALTER COLUMN "messages" DROP NOT NULL;--> statement-breakpoint
+ALTER TABLE "latitude"."provider_logs" ALTER COLUMN "duration" DROP NOT NULL;--> statement-breakpoint
+ALTER TABLE "latitude"."provider_logs" ALTER COLUMN "generated_at" DROP NOT NULL;
```

Comment:
Why we're removing all this not null. I guess is because you want to handle the case of manually uploaded logs. But somehow looks like is making this model more weak no?

Reply:
yup but there's no other way that isn't more painful

Reply:
ufff ya